game.html

```diff
@@ -17,8 +17,12 @@
     </audio>
     <body>
         <div id="gameModeDialog" title="Game Mode Selection" class="dialog">
-            <input type="radio" id="standardRadioButton" name="mode" checked>Standard Mode<br>
-            <input type="radio" id="freeColorRadioButton" name="mode">Loosey Goosey Mode<br>
+            <input type="radio" id="standardRadioButton" name="mode" checked>
+            <label for="standardRadioButton">Standard Mode</label>
+            <br>
+            <input type="radio" id="freeColorRadioButton" name="mode">
+            <label for="freeColorRadioButton">Loosey Goosey Mode</label>
+            <br>
```

Comment:
I was annoyed that clicking the text wouldn't select the radio option. The text needs to be in a `<label for>` for that to work.